### ISSUE TITLE
 Can Tikv be used as a metadata storage system?

### ISSUE THREAD

**Issue by melin** [tikv/tikv#11950]
Snowflale, firebolt databases use foundationdb to store metadata，CCB MPP database also uses FDB,：https://mp.weixin.qq.com/s/FVnrDvnz0SZjhuJ-hugqCQ
Can TIKV be applied in such a scenario? Is there any comparative test data on performance? Our products are also being selected, and we hope to have some data to support the selection

**Comment by melin** [comment]
@BusyJay

**Comment by zz-jason** [comment]
Yes, TiKV can be used as a meta store. For example, JuiceFS use TiKV to store the metadata of the file system. You can refer  to [this post](https://en.pingcap.com/blog/get-started-with-juicefs-using-tikv/) about the performance comparations.

**Comment by zz-jason** [comment]
could you tell us more about your use scenarios, for example, what's the service looks like, what's the TPS/QPS and latency requirement about the meta store, does it require ACID transaction support?

**Comment by melin** [comment]
> could you tell us more about your use scenarios, for example, what's the service looks like, what's the TPS/QPS and latency requirement about the meta store, does it require ACID transaction support?

Stores database and table metadata instead of Hive MetaStore. Similar to the aws glue
